cobalt/h5vcc/h5vcc_runtime.cc

```diff
@@ -87,8 +87,8 @@ void H5vccRuntime::TraceMembers(script::Tracer* tracer) {
 void H5vccRuntime::OnEventForDeepLink(const base::Event* event) {
   std::unique_ptr<base::DeepLinkEvent> deep_link_event(
       new base::DeepLinkEvent(event));
-  if (base::MessageLoop::current() != message_loop_) {
-    message_loop_->task_runner()->PostTask(
+  if (!task_runner_->RunsTasksInCurrentSequence()) {
```

Comment:
Shouldn't there be a validity check for `task_runner` to be live here ?

Reply:
Check added.